### PR TITLE
Fixes #34985 - Extend windows templates

### PR DESCRIPTION
**Pull request by SimonLorentz**
To test this changes, deploy windows hosts with ansible and/or puppet enabled. Use puppet with a separate puppet environment, not production. The new hosts should be fully updated, if they have internet connection.

### CODE REVIEW

File: app/views/unattended/provisioning_templates/finish/windows_default_finish.erb [hunk 1]
```diff
@@ -44,12 +41,10 @@ description: |
...
   set ctr=0
   set nettimeout=10
+<% end -%>
 
-  (echo Updating time)
-  (sc config w32time start= auto)
-  sc start w32time
-  ::ipconfig /renew
-
+  <%= snippet 'Windows network simon' %>
```
Comment (evgeni) [inline app/views/unattended/provisioning_templates/finish/windows_default_finish.erb]: Simon who? ;-)

File: app/views/unattended/provisioning_templates/finish/windows_default_finish.erb [hunk 2]
```diff
@@ -70,63 +65,75 @@ description: |
       <% end %>
     <% end %>
   <% end %>
+  
+  powershell /c "powercfg /setactive 8c5e7fda-e8bf-4a96-9a85-a6e23a8c635c"
```
Comment (evgeni) [inline app/views/unattended/provisioning_templates/finish/windows_default_finish.erb:69]: what is this UUID?

**Comment by evgeni** [comment]
ok to test

File: app/views/unattended/provisioning_templates/finish/windows_default_finish.erb (hunk 1 above)
Comment (SimonLorentz) [reply app/views/unattended/provisioning_templates/finish/windows_default_finish.erb]: right, this is wrong I will correct it, thanks

File: app/views/unattended/provisioning_templates/finish/windows_default_finish.erb (hunk 2 above)
Comment (SimonLorentz) [reply app/views/unattended/provisioning_templates/finish/windows_default_finish.erb:69]: this is the default Power Scheme GUID for High performance

File: app/views/unattended/provisioning_templates/finish/windows_default_finish.erb [hunk 3]
```diff
@@ -8,31 +8,28 @@ oses:
...
-  - computerDomain: domain.com # domain to join
-  - machinePassword: used for unsecure domain join. needs precrated computer object (New-ADComputer)
-  - foremanDebug: false
-  - skip-puppet-setup: boolean (default=false)
-
-  Information about unsecure domain join
-  https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/add-computer?view=powershell-5.1#example-9--add-a-computer-to-a-domain-using-predefined-computer-credentials
--%>
+# Parameters are expected to be set in Foreman (globally or per group/host)
+params:
```
Comment (evgeni) [inline app/views/unattended/provisioning_templates/finish/windows_default_finish.erb:12]: Where/how is this rendered? `description` ends up in the Edit view, but I don't see a place where we'd render `params`?

File: app/views/unattended/provisioning_templates/finish/windows_default_finish.erb [hunk 4]
```diff
@@ -8,31 +8,28 @@ oses:
...
+- ntpServer: time.windows.com,other.time.server
+- domainAdminAccount: joinuser@domain.com # please do not use the domain administrator
+- domainAdminAccountPasswd: Password for the domain Admin account
+- computerOU: OU=Computers,CN=domain,CN=com # Place the computer account in specified Organizational Unit
+- computerDomain: domain.com # domain to join
+- machinePassword: used for unsecure domain join. needs precrated computer object (New-ADComputer)
+%>
+<%#
+# Information about unsecure domain join
+# https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/add-computer?view=powershell-5.1#example-9--add-a-computer-to-a-domain-using-predefined-computer-credentials
```
Comment (evgeni) [inline app/views/unattended/provisioning_templates/finish/windows_default_finish.erb:25]: this is now a comment in the template itself, vs part of the description as it was before. Is that intentional?

**Comment by evgeni** [comment]
We're currently lacking people in the core team who would be knowledgeable about Windows deployments, so it would be cool if someone from @ATIX-AG (or any other Foreman Windows user) would review the "this works fine for windows" part please.

**Comment by nadjaheitmann** [comment]
We would like to close this PR in favor of https://github.com/theforeman/foreman/pull/9732 . Can someone help?

**Comment by evgeni** [comment]
Closed on request of @nadjaheitmann

**Pull request closed without merging**